ts/packages/anchor/src/program/namespace/methods.ts

```diff
@@ -161,6 +166,21 @@ export class MethodsBuilder<IDL extends Idl, I extends AllInstructions<IDL>> {
     >;
   }
 
+  // TODO: Remove the hard-coded discriminators once we have a JS package for Transfer Hook
```

Comment:
I think the main Anchor package shouldn't depend on any SPL package so hardcoding should be fine here.